### PR TITLE
Add docs for Lux Meter

### LINKED ISSUE #66: Instructions for Lux Meter
**Describe the issue**

There are no instructions for Lux Meter. There should be instructions for how to use Lux Meter 
**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by MuhammadWasif**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **Related Issue**

Fix #66 

* **What changes have you introduced?**
Added docs for Lux Meter with screenshots
* **Preview / Steps to verify your work**:
* Go to Lux Meter docs

### CODE REVIEW

File: tutorials/luxmeter.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+# Lux Meter
 
-What is a Luxmeter
------------------------
\ No newline at end of file
+
+## What is Lux Meter
+Lux Meter is used to measure light intensity
+
+## Experiment for Accelerometer
```
Comment (NishithP2004) [inline tutorials/luxmeter.md]: Why is it put accelerometer?

Comment (MuhammadWasif) [reply tutorials/luxmeter.md]: Fixed...!

File: tutorials/luxmeter.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* [PSLab Android App](https://play.google.com/store/apps/details?id=io.pslab&hl=en_US)
+
+### Procedure
+* Open PSLab Android App
+* Scroll down to the **Lux Meter**
+![Screenshot](/_static/lux_image_1.jpg)
+
+* You will see a graphical representation as well as numerical values of lux along with a meter 
+* Bring your device near different lights. For example, yellow light or white light etc.
+
```
Comment (Sid911) [inline tutorials/luxmeter.md]: I guess you should mention where the **Ambient light sensor is present** in smartphones as usually they are hidden off in the from... pointing rear camera won't really help in this case 😇

Comment (Sid911) [reply tutorials/luxmeter.md]: an image of sensor array on a smartphone's front _head_ would be better

**Comment by MuhammadWasif** [comment]
@mariobehling Please review  😊

**Comment by MuhammadWasif** [comment]
@cweitat please merge.

**Pull request merged**